app/src/main/java/org/xbmc/kore/service/NotificationObserver.java

```diff
@@ -224,51 +230,83 @@ private void notifyPlaying(PlayerType.GetActivePlayersReturnType getActivePlayer
-                .setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
+                new NotificationCompat.Builder(service, NOTIFICATION_CHANNEL);
+
+        builder.setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
                 .setSmallIcon(smallIcon)
                 .setShowWhen(false)
                 .setOngoing(true)
-                .addAction(rewindIcon, service.getString(R.string.rewind), rewindPendingIntent) // #0
-                .addAction(playPauseIcon, service.getString(R.string.play), playPausePendingIntent)  // #1
-                .addAction(ffIcon, service.getString(R.string.fast_forward), ffPendingIntent)     // #2
-                .setStyle(new androidx.media.app.NotificationCompat.MediaStyle()
-                              .setShowActionsInCompactView(0, 1, 2))
                 .setContentIntent(remoteStartPendingIntent)
                 .setContentTitle(title)
                 .setContentText(underTitle);
 
+        if (getItemResult.type.equals(ListType.ItemsAll.TYPE_EPISODE)
+                || getItemResult.type.equals(ListType.ItemsAll.TYPE_MOVIE)
+        ) {
+            builder.addAction(rewindIcon, service.getString(R.string.rewind), rewindPendingIntent) // #0
+                    .addAction(jumpBackIcon, service.getString(R.string.rewind), jumpBackPendingIntent) // #1
+                    .addAction(playPauseIcon, service.getString(R.string.play), playPausePendingIntent)  // #2
+                    .addAction(jumpForwardIcon, service.getString(R.string.fast_forward),
+                            jumpForwardPendingIntent) // #3
+                    .addAction(ffIcon, service.getString(R.string.fast_forward), ffPendingIntent)   // #4
+                    .setStyle(new androidx.media.app.NotificationCompat.MediaStyle()
+                            .setShowActionsInCompactView(1, 2, 3));
+        } else {
+            builder.addAction(rewindIcon, service.getString(R.string.rewind), rewindPendingIntent) // #0
+                    .addAction(playPauseIcon, service.getString(R.string.play), playPausePendingIntent)  // #1
+                    .addAction(ffIcon, service.getString(R.string.fast_forward), ffPendingIntent)  // #2
+                    .setStyle(new androidx.media.app.NotificationCompat.MediaStyle()
+                            .setShowActionsInCompactView(0, 1, 2));
+        }
+
+        // Creates Media Style with MediaSession Token required to display
+        // media controls below quick settings in Android 11 and above.
+        if (Build.VERSION.SDK_INT >= 21) {
+            final MediaSessionCompat mediaSession = new MediaSessionCompat(this.service,
+                    "MediaService");
+            final MediaStyle mediastyle = new androidx.media.app.NotificationCompat.MediaStyle()
+                    .setMediaSession(mediaSession.getSessionToken());
+            builder.setStyle(mediastyle
+                    .setShowActionsInCompactView(1, 2, 3));
+        }
```

Comment:
I had to comment this block to make the notification appear. This is probably interfering with the previous code, as it is again setting the style on the builder, and at least my Android version doesn't like that.
This should be better integrated with the previous code somehow.